File: OnJeong_BE/src/main/java/com/a503/onjeong/domain/user/dto/UserDTO.java
package com.a503.onjeong.domain.user.dto;

import lombok.Builder;
import lombok.Data;

@Data
public class UserDTO {
    private Long userId;
    private String name;
    private String phoneNumber;
    private String profileUrl;
    @Builder
    public UserDTO(
            Long userId,
            String name,
            String phoneNumber
    ) {
        this.userId = userId;
        this.name = name;
        this.phoneNumber = phoneNumber;
    }

    public void setProfileUrl(String profileUrl) {
        this.profileUrl = profileUrl;
    }
}

File: OnJeong_FE/app/src/main/java/com/a503/onjeong/domain/mypage/dto/UserDTO.kt
package com.a503.onjeong.domain.mypage.dto

import lombok.Builder
import lombok.Data


@Data
class UserDTO @Builder constructor(
    private val userId: Long,
    private val name: String,
    val phoneNumber: String,
    val profileUrl: String
) {
    fun getName(): String {
        return name
    }

    fun getUserId(): Long {
        return userId
    }
}

